### ISSUE TITLE
U.S data exaggerates infections by 1000

### ISSUE THREAD

**Issue by urimerhav** [CSSEGISandData/COVID-19#559]
Looks like the current total number for US infections in this repo exceeds 2000, but [other authoritative sources](https://www.worldometers.info/coronavirus/) report 1300 cases.

The rest of the numbers seem to line up with other sources.

**Comment by urimerhav** [comment]
Way to see the problem: group data by country and sum.

See this notebook that does this and display by country.

https://colab.research.google.com/drive/1NvY5UfxgEsOZkBHvv82S1KaTkSmyeino#scrollTo=pjAXPBZqXXOM

**Comment by JiPiBi** [comment]
@urimerhav 
I read your interesting notebook , but IMO for US , you had to filter out Province with a comma .

**Comment by urimerhav** [comment]
@JiPiBi 
What do you mean by filter out province with a comma? If you could share a fixed version, or fruther clarify what you mean with code, I'd really appreciate it!

**Comment by urimerhav** [comment]
Do you mean to say there's double counting because records are reported at both the state and county level starting at some point in time, and that the way to tell when a piece of data is county data, us that we have a comma in the string?

If so, that's great to learn as a workaround. I still think having double-counted data is asking for trouble. I know there's another issue around provinces. So I guess you can consider this issue to be +1 for reporting only at county level and letting people aggregate to state level on their own.

**Comment by JiPiBi** [comment]
I mean that from yesterday there is a double reporting for US, numerous issues about that.  The new Provinces  created yesterday seem to be the only usable to aggregate the US values

**Comment by urimerhav** [comment]
How do I filter out all the non-province data exactly to get an accurate count?

**Comment by dompazz** [comment]
I'm looking at trying to figure out the right hoops to jump through to get a correct time series for the US.  I don't have the correct values through time to compare to.  If someone has that, can they please post that?

**Comment by dompazz** [comment]
Here's what I have.  I use SAS but this should be able to be changed as needed

Confirmed cases are read into a table and transposed to fit into:

> create table COVID.CONFIRMED
>   (
>    date num format=DATE9.,
>    Province_State char(28) format=$28. informat=$28.,
>    Country_Region char(30) format=$30. informat=$30.,
>    Lat num format=BEST12. informat=BEST32.,
>    Long num format=BEST12. informat=BEST32.,
>    confirmed num format=BEST12.
>   );

Then I filter and sum as:

> create table _temp as
> select * from covid.confirmed;
> 
> delete from _temp
> 	where country_region = 'US'
> 	  and index(Province_State,',')
> 	  and date >= "10MAR2020"d;
> 
> create table total_confirmed as
> select date,
> 	   country_region,
> 	   sum(confirmed) as confirmed
> 	from _temp;

The `index()` function returns the location of a character in a string, returning 0 if not found.  This filters the city/state values for the last two days.

**Comment by augix** [comment]
The numbers are roughly right if only use the rows with complete county and state info. What's the reason behind the duplication of records?

**Comment by joelaaronseely** [comment]
That's right - there are separate records with non-zero data starting on 3/10 for US states with blank "Province/State" fields.  For example "Washington" in the Province/Field shows 267 on 3/10 and 366 on 3/11 but zero on all dates before this.  But there are also records with individual Province state data, for example, "Kitsap, WA" (and others).  This implies a double-counting going on.

**Comment by zac-garland** [comment]
are there plans to fix this? I think the main issue is the misrepresentation of USA data. Even on your dashboard, it shows 0 cases in Colorado, although there are ~72, while the county level data shows 0 as of today, and the state data shows 45? 

Alternatively, maybe make a note so we aren't spreading misinformation?

**Comment by urimerhav** [comment]
I think we're better off kicking out all the "province, state" data into a separate historical csv for backwards compatibility, and have the fresh csv show only state data. Either way, having double-counted items is going to mislead most people who use this dataset in the future.
